chem/KPP/configure_kpp

```diff
@@ -80,6 +80,9 @@ echo " configure_kpp, settings:"
 if test -e "${FLEX_LIB_DIR}/libfl.a" ; then
 echo location of flex library: ${FLEX_LIB_DIR}/libfl.a
 
+elif test -e "${FLEX_LIB_DIR}/libfl.so" ; then
+echo use shared flex library: ${FLEX_LIB_DIR}/libfl.so
+
 else
           echo  No libfl.a in ${FLEX_LIB_DIR} 
```

Comment:
echo message should probably be updated to reflect check for `.so` library as well

Reply:
@islas - see latest commit - does this address your concern?

Reply:
Yes, thank you!